custom_components/powercalc/data/signify/929003052501_01/model.json

```diff
@@ -0,0 +1,16 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "master"
+    },
+    "name": "Hue Ensis",
+    "standby_power": 0.4,
+    "aliases": [
+       "929003052501_02"
```

Comment:
Can you set directory name to 4090330P9 and add the zigbee ID's as aliases as listed here:
https://zigbee.blakadder.com/Philips_4090330P9.html